crates/web-sys/webidls/enabled/Element.webidl

```diff
@@ -263,3 +263,18 @@ partial interface Element {
   [NeedsCallerType]
   undefined requestPointerLock();
 };
+
+interface mixin HTMLOrForeignElement {
+  [SameObject] readonly attribute DOMStringMap dataset;
+
+  [CEReactions, SetterThrows, Pure] attribute boolean autofocus;
+  [CEReactions, SetterThrows, Pure] attribute long tabIndex;
+  [Throws, NeedsCallerType] undefined focus(optional FocusOptions options = {});
+  [Throws] undefined blur();
+};
```

Comment:
Sure we can do that for now I considered it anyway. There was already https://github.com/whatwg/html/pull/5248 and I thought it would be more accurate since it is not either HTML or SVG (and I guess that sooner or later that rename will be done, but we can adjust that when it's official).

Reply:
I see.
Yeah, I would prefer not to be pre-emptive, in my experience so far things are extremely fluid in this place.